components/ButtonBase.qml

```diff
@@ -49,7 +49,7 @@ Rectangle {
             else {
                 buttonContainer.opacity = 0.3
                 buttonContainer.enabled = false
-                buttonContainer.close();
+                //buttonContainer.close();
```

Comment:
Why is close() no longer required?

Reply:
Hi, the problem is that it is really annoying when you start a movie with the remote and want to pause it after 5 minutes.

When the remote wakes up from standby the entity is closed because it is not connected anymore and you have to reopen it again, go to the transport view and then press pause.
It is really inconvenient for a remote.

Reply:
That makes sense.

Reply:
I will remove it completely then instead of commenting it